boot.sh

```diff
@@ -32,7 +32,7 @@ readonly DOCKER_CLIENT_LATEST="/opt/docker/docker"
 readonly BOOT_WAIT_TIME=10
 readonly SWAP_FILE_PATH="/root/.__sh_swap__"
 
-export DOCKER_VERSION=""
+export DOCKER_VERSION_INSTALLED=""
```

Comment:
if DOCKER_VERSION already exists from being defined in the node script, do we even need to set it again? is this version going to be different from the one set in the node script?